lib/config/rules/stylistic-issues.js

```diff
@@ -364,9 +364,6 @@ module.exports = {
       'error',
       'never',
       {
-        blocks: 'never',
-        classes: 'never',
-        switches: 'never',
         allowSingleLineBlocks: true
       }
     ],
```

Comment:
I think instead of removing this rule, it might be more appropriate to update it to match the new [rule syntax](https://eslint.org/docs/rules/padded-blocks) like below:

```
'padded-blocks': [
  'error',
  {
    blocks: 'never',
    classes: 'never',
    switches: 'never',
  }, {
    allowSingleLineBlocks: true,
  }
],
```

I tested this with v7.4.0 and it seems to work. Thoughts?

Reply:
I must have missed that new syntax. Thanks for letting me know about it.

I'd be happy to update this to use the newer syntax so that we maintain the same set of rules as before. Do you know if this change is compatible with eslint v6 by chance?

Reply:
It looks like it would be: https://eslint.org/docs/6.0.0/rules/padded-blocks

Wouldn't hurt to test though. I'll see if I can do that quick.

Reply:
I've always wondered how to get access to docs in older versions 😆 . Thanks for pointing me to that! Seems v7 got stricter about validating the rule configuration, so I'm not surprised we haven't seen this issue before.

Reply:
Addressed in a561c61

Reply:
Good news - seems to work in v6.8.0 just fine!